scripts/std-lib-regression.sh

```diff
@@ -73,11 +73,16 @@ export RUSTC="$KANI_DIR/target/kani/bin/kani-compiler"
 # Compile rust to iRep
 $WRAPPER cargo build --verbose -Z build-std --lib --target $TARGET
 
+# Runs a command, but suppresses output unless it fails
+function go {
+  OUTPUT=$("$@" 2>&1) || (echo "$OUTPUT" && exit 1)
+}
+
 # Generate goto-program. This will make sure the representation is well formed.
 cd target/${TARGET}/debug/deps
 for symtab in *.symtab.json; do
     echo "======== File: $symtab"
-    symtab2gb ${symtab} --out ${symtab}.out
+    go symtab2gb ${symtab} --out ${symtab}.out
```

Comment:
wait... can't we just remove these calls now?

Reply:
Well you tell me :P

I thought we chatted and this test was still necessary because it tested the gotoc generation from stdlib, and the new sysroot stuff only generated MIR.

Reply:
The test is still required. But we no longer need to explicit invoke symtab in this script since kani-compiler already does.

Reply:
Sorry for the confusion